h.helm.md

```diff
@@ -4,122 +4,145 @@
 
 ## Helm in K8s
 
-### Creating a basic Helm chart
+### Creating a Basic Helm chart
```

Comment:
why capitalize Basic?

Reply:
Ah, this doesn't need to be capitalized. I'll make a change.